packages/webview_flutter/webview_flutter_android/lib/src/android_webview_controller.dart

```diff
@@ -495,6 +537,25 @@ class AndroidWebViewController extends PlatformWebViewController {
     _onGeolocationPermissionsShowPrompt = onShowPrompt;
     _onGeolocationPermissionsHidePrompt = onHidePrompt;
   }
+
+  /// Sets the callbacks that are invoked when the host application wants to
+  /// show or hide a custom view.
+  ///
+  /// The most common use case these methods are invoked a video element wants
+  /// to be displayed in fullscreen.
+  ///
+  /// The [onShowCustomView] notifies the host application that web content
```

Comment:
I think this should also include that `After this call, web content will no longer be rendered in the `WebViewWidget`, but will instead be rendered in the custom widget.`